sklearn/metrics/tests/test_score_objects.py

```diff
@@ -283,7 +289,10 @@ def test_classification_scores():
     clf.fit(X_train, y_train)
 
     for prefix, metric in [('f1', f1_score), ('precision', precision_score),
-                           ('recall', recall_score)]:
+                           ('recall', recall_score),
+                           ('jaccard_similarity',
+                            partial(jaccard_similarity_score,
+                                    average='binary'))]:
```

Comment:
Because it's a bit strange, it might be good to test that this scorer actually works... I don't think we even have smoke tests for all the scorers, but we probably should...

Reply:
> I don't think we even have smoke tests for all the scorers, but we probably should...

Do you mean that I should add it for just jaccard similarity with average='macro', average='micro', average='weighted', ... or for all possible scorers as well (i.e. including f1, precision...).

I haven't written smoke tests before, but I do see their presence by grepping the code base.

Reply:
Wouldn't this https://github.com/gxyd/scikit-learn/blob/0b507aa2756132c63670acf6539f5dd7370c6f94/sklearn/metrics/tests/test_score_objects.py#L312 suffice as a smoke test? Since it just takes 'jaccard_similarity' with average='binary'